package.json

```diff
@@ -16,6 +16,7 @@
     "@material-ui/core": "3.9.3",
     "@material-ui/icons": "3.0.2",
     "@octokit/rest": "16.28.7",
+    "@testing-library/react": "9.1.0",
```

Comment:
I am adding this `@testing-library/react` for experimenting using the render method.